<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="genLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пол</string>
   </property>
  </widget>
  <widget class="QLabel" name="powLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сила</string>
   </property>
  </widget>
  <widget class="QLabel" name="agilLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ловкость</string>
   </property>
  </widget>
  <widget class="QLabel" name="intelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Интеллект</string>
   </property>
  </widget>
  <widget class="QLabel" name="luckLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Удача</string>
   </property>
  </widget>
  <widget class="QLabel" name="resLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Всего</string>
   </property>
  </widget>
  <widget class="QLabel" name="hiLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Привет, укажи свое имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="classLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Класс</string>
   </property>
  </widget>
  <widget class="QLabel" name="xpLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ХП</string>
   </property>
  </widget>
  <widget class="QLabel" name="mpLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>МП</string>
   </property>
  </widget>
  <widget class="QLabel" name="attLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Атака</string>
   </property>
  </widget>
  <widget class="QLabel" name="defLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Защита</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Сгенерировать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="intelEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="agilEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="powEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="luckEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="xpEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mpEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="attEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="defEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="mButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>М</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="wButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ж</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Шахматы</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Бойцы</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>*показатели всех основных характеристик не должны превышать 15</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="baseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---Основные хар-ки---</string>
   </property>
  </widget>
  <widget class="QLabel" name="ostLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
